src/components/pages/manage/productions/create/index.js

```diff
@@ -235,6 +242,7 @@ const Create = () => {
                   maxWidth="43rem"
                   label={t('productions.create.production_name')}
                   emptyLabel={t('productions.create.no_productions')}
+                  minLength={minSearchLength}
                   onInputChange={throttle(handleInputSearch, 275)}
```

Comment:
Looks good, the only improvement I see is that throttle is still called a few times while we don't want to.

So maybe we could write it like this:

```javascript
onInputChange={handleInputSearch}
```

And 

```javascript
const handleInputSearch = useCallback((searchTerm) => {
  const trimmedSearchTerm = searchTerm.toString().trim();
  if (trimmedSearchTerm.length < minSearchLength) return;
  return throttle(() => {
    setSelectedProductionId(undefined);
    setSearchInput(trimmedSearchTerm);
  }, 275);
})
```

Reply:
Is it not more clear if the throttle stays in the same place?

Reply:
Yes it's clearer. Just leave it as it is. I'll approve